github-dark.css

```diff
@@ -2733,6 +2737,19 @@
   .zhc-readonly-banner {
     background: #182030 !important;
   }
+  .zhc-progress-bar {
+    background: #333 !important;
+  }
+  .zhc-progress-bar > .zhc-progress-bar--completed-issues {
+    background: #163 !important;
+  }
+  .zhc-item-table__header {
+    color: #e2e2e2 !important;
+    background: #202020 !important;
+  }
+  .zhc-item-table__cell__issue-number {
+    color: #7b7b7b !important;
+  }
```

Comment:
While we are keeping the ZenHub definitions separate from the main page definitions, please try to combine these new additions with other ZenHub definitions... with same, or nearly-similar colors. Thanks!